### PR TITLE
Updates todo page to show only items in your 'squad'

### PR DESCRIPTION
**Pull request by adambray**
Does this by updating the get_due method to search for memberships with
tags in common with your admin memberships.

### CODE REVIEW

**Comment by andrewsunglaekim** [comment]
Can we merge [tagging_ux](https://github.com/ga-dc/garnet/pull/315)? Would make it easier to see the diff for these PR's

**Comment by adambray** [comment]
@andrewsunglaekim fine with that once that PR gets one more pair of :eyeglasses:

**Comment by jshawl** [comment]
permission to rebase and force push to show the updated diff after https://github.com/ga-dc/garnet/commit/868367c1f028108595f429f29ab949b0ef6ecefe ?

**Comment by adambray** [comment]
sure!

File: app/controllers/dashboards_controller.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ def to_do
 
     redirect_to current_user unless @user.is_admin_of_anything?
 
-    @due_submissions = @user.get_due("submissions")
-    @due_attendances = @user.get_due("attendances")
+    @due_submissions = @user.get_todo(Submission)
+    @due_attendances = @user.get_todo(Attendance)
```
Comment (jshawl) [inline app/controllers/dashboards_controller.rb]: have a feeling this should be a symbol a la `has_many` and `belongs_to`. is there any difference between doing something like this and `@user.get_todo :submission`

**Comment by jshawl** [comment]
:+1: just comments for a future `TODO` or optimization but otherwise good to merge

File: app/controllers/dashboards_controller.rb (hunk 1 above)
Comment (adambray) [inline app/controllers/dashboards_controller.rb]: Keeps the actual method implementation (of `get_todo`) cleaner. No need to take the symbol and do `model.capitalize.constantize` etc.

**Pull request merged**